Polygon_mesh_processing/include/CGAL/Rigid_triangle_mesh_collision_detection.h

```diff
@@ -537,11 +537,9 @@ class Rigid_triangle_mesh_collision_detection
       std::vector<std::size_t> cc_ids(num_faces(tm));
 
       // face index map
-      typedef typename CGAL::GetFaceIndexMap<TriangleMesh, NamedParameters>::type Fid_map;
 
-      Fid_map fid_map =
-        parameters::choose_parameter(parameters::get_parameter(np, internal_np::face_index),
-                            get_const_property_map(boost::face_index, tm));
+      auto fid_map =
+        CGAL::Polygon_mesh_processing::get_initialized_face_index_map(tm, np);
```

Comment:
Are there really only places in PMP where this changes stuff? IIRC, SMP also wants nicely initialized index maps too, no?

Reply:
I've seen traces of index_maps in shortest_path, but the whole class is templated by them, and there are no named parameters. It would require a big effort to make it use this mechanism, and the small feature that goes with it. I'm not doing it. At least not in that PR.